terraform/variables/staging-facil.tfvars

```diff
@@ -45,6 +44,12 @@ ingestors = {
     }
   }
 }
+cluster_settings = {
+  initial_node_count : 2
+  min_node_count : 2
```

Comment:
Our node pools are regional/multi-zonal, so all these values are per-AZ. I think one node per AZ will be enough for staging. Can we try `initial_node_count = 1` and `min_node_count = 1`?